toolchain/check/import_ref.cpp

```diff
@@ -1987,6 +2018,9 @@ static auto TryResolveTypedInst(ImportRefResolver& resolver,
       GetLocalParamPatternsId(resolver, import_function.param_patterns_id);
   new_function.return_slot_pattern_id = GetLocalReturnSlotPatternId(
       resolver, import_function.return_slot_pattern_id);
+  new_function.call_params_id =
```

Comment:
The comment on this member currently says "This is not populated on imported functions, because it is relevant only for a function definition", but that's no longer accurate after this change.